recipes/pytest-monitor/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "pytest-monitor" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: //https://pypi.io/packages/source/p/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Seems like there are two too much slashes
```suggestion
  url: https://pypi.io/packages/source/p/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

---

message crossing :) Thanks !